etc/packaging/appimages/viam-server-aarch64.yml

```diff
@@ -32,16 +34,8 @@ AppDir:
     - sourceline: deb [trusted=yes] http://deb.debian.org/debian bullseye main
     - sourceline: deb [trusted=yes] http://deb.debian.org/debian-security bullseye-security main
     - sourceline: deb [trusted=yes] http://deb.debian.org/debian bullseye-updates main
-    - sourceline: deb [trusted=yes] https://us-apt.pkg.dev/projects/static-file-server-310021 bullseye main
-    - sourceline: deb [trusted=yes] http://archive.raspberrypi.org/debian/ bullseye main
     include:
     - bash
-    - libnlopt0
-    - libx264-160
-    - libtensorflowlite2
-    - libpigpio1
-    - libopus0
-    - libjpeg62-turbo
```

Comment:
The appimages now include the static binary by default, making them smaller and faster to run/build. This is an interim step, as I expect appimages to go away entirely for viam-server itself before too much longer.